src/routes/routes.ts

```diff
@@ -0,0 +1,16 @@
+const appRoutes = {
+  AnalysisLatest: {
+    path: '/analysis/latest',
```

Comment:
Path property unnecessary if no other properties are planed

Reply:
You're right.
First wanted to add some other properties aswell but I think that your proposal works better. I don't have any other properties planned.